assets/pages/regular-contributions/components/regularContributionsPayment.jsx

```diff
@@ -104,6 +106,7 @@ function RegularContributionsPayment(props: PropTypes, context) {
       props.contributionType,
       props.dispatch,
       'stripeToken',
+      'stripe',
```

Comment:
I don't feel too strongly about this, but it seems odd to have to tell the Stripe button that it's using Stripe. One way around this would be to have `postCheckout` curried, partially apply it with all the other data, and then call that the result of that partial with the payment method. However, I don't think that's super-important; it works fine the way it is.

Reply:
Same is true for 'stripeToken' on the line above